etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/validation/ValidationRule.java

```diff
@@ -28,12 +27,19 @@ public ValidationRule(
     }
 
     @Override
-    public void runRule(FhirResource<?> resource) {
+    public void runRule(HealthData<?>... resource) {
+
+        if (resource.length != 1) {
```

Comment:
Should this go into the parent rule class and call super each time? It would put all the code in one place

Reply:
Depending on the Rule inheritor implementation, it will have different number of HealthData objects. AssertionRule.runRule takes two HealthData objects, for example